docker-compose.yaml

```diff
@@ -13,19 +13,22 @@ services:
             - DD_APM_NON_LOCAL_TRAFFIC=true
             - DD_API_KEY=${DD_API_KEY}
```

Comment:
What happens when there is no api key?

Reply:
The API Key will be passed empty and HASH will skip logging with Datadog. Nothing will break.